apps/src/studio/bigGameInfo.js

```diff
@@ -0,0 +1,153 @@
+var Direction = require('./constants').Direction;
+
+var BigGameInfo = function (studio) {
```

Comment:
Yeah, I started removing it and then I decided I liked BigGameInfo better than BigGame. I think BigGameLogic might be better.